released/packages/coq-autosubst-ocaml/coq-autosubst-ocaml.1.0.0/opam

```diff
@@ -0,0 +1,55 @@
+opam-version: "2.0"
+synopsis: "A code generator for substitution boilerplate"
+description: """
+  This is an OCaml reimplementation of the Autosubst 2 code generator by Stark, Schäfer and Kaiser,
+  which was the main focus of Stark's doctoral dissertation. Autosubst 2 in turn is based on 
+  previous work on Autosubst by Schäfer, Tebbi and Smolka.
+
+  If you ever were in the situation of looking at metatheorems of languages modelled in Coq 
+  (e.g. progress and preservation of lambda calculi) and were bothered by the tediousness of 
+  formalizing substitution of de Bruijn indices again, this tool might be for you. Autosubst is a 
+  tool that allows you to quickly generate boilerplate code to handle substitutions in languages 
+  with binders.
+
+  The output is Coq source code that contains
+
+  - an implementation of the language via (mutual) inductive types and de Bruijn indices for variables,
+  - definitions for capture avoiding substitution and renaming on de Bruijn indices,
+  - lemmas about the behavior and interaction of renaming and substitution,
+  - and a tactic to solve assumption-free substitution equations.
+""" 
+
+homepage: "https://github.com/uds-psl/autosubst-ocaml"
+dev-repo: "git+https://github.com/uds-psl/autosubst-ocaml.git"
+bug-reports: "https://github.com/uds-psl/autosubst-ocaml/issues"
+maintainer: "s8addapp@stud.uni-saarland.de"
+doc: "https://www.ps.uni-saarland.de/~dapprich/bachelor.php"
+authors: [
+  "Adrian Dapprich"
+]
+license: "MIT"
+
+depends: [
+  "ocaml" { >= "4.09" }
+  "coq" { = "8.13.1" }
```

Comment:
```suggestion
  "coq" {>= "8.13" & < "8.14"}
```
Are you sure the package only works with Coq 8.13.1? Due to Coq's compatibility policy, nearly all plugins should work across all minor versions for a major release (e.g., 8.13.2 works if it works on 8.13.0).

Reply:
yes, I think I just used the output of opam list directly. I will check it